doc-content/enterprise-only/weblogic/wls-data-source-create-proc.adoc

```diff
@@ -7,6 +7,21 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
 * The JDBC drivers that you want to use to create database connections are installed on all servers on which you want to deploy the data source. Some JDBC drivers are installed with {WEBLOGIC}, such as WebLogic-branded Data Direct JDBC drivers for DB2, Informix, MS SQL Server, and Sybase. For more information about JDBC drivers, see https://docs.oracle.com/middleware/12213/wls/JDBCA/third_party_drivers.htm#JDBCA231[Using JDBC Drivers with WebLogic Server] in the Oracle Help Center.
 
 .Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
```

Comment:
Go to the <downloads> page. See pg 347 in the style guide.

Reply:
Good point, but this is the format that we use throughout our docs when referring to the CP. I'll take it up with Stetson (his formulation :-))